.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug, needs triage
+assignees: ''
+
+---
+
+## Describe the bug
+
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '...'
```

Comment:
"Click on" and "Scroll down" probably aren't super applicable to these samples, could you update the wording here?